source/_localCaptioner/modelManager.py

```diff
@@ -0,0 +1,686 @@
+# -*- coding: UTF-8 -*-
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2025 NV Access Limited, tianze
+# This file may be used under the terms of the GNU General Public License, version 2 or later, as modified by the NVDA license.
+# For full terms and any additional permissions, see the NVDA license file: https://github.com/nvaccess/nvda/blob/master/copying.txt
+import wx
+import threading
+from typing import List, Optional
+import winsound  # Windows sound API for audio notifications
+
+# Assume modelDownloader.py is in the same directory
+try:
+	from .modelDownloader import downloadModelsMultithreaded, ensureModelsDirectory
+except ImportError:
+	from modelDownloader import downloadModelsMultithreaded, ensureModelsDirectory
+
+
+try:
+	from logHandler import log
+	import addonHandler
+
+	addonHandler.initTranslation()
+except:
+	_ = format
+
+
+class AdvancedSettingsDialog(wx.Dialog):
```

Comment:
Please integrate the settings dialog as a new panel in settingsDialogs which makes it directly part of the NVDA settings dialog